dags/common/ftp_service.py

```diff
@@ -0,0 +1,79 @@
+import ftplib
+import io
+import os
+import re
+import traceback
+
+from common.exceptions import DirectoryNotFoundException, NotConnectedException
+from common.utils import append_file_if_not_in_excluded_directory, walk_ftp
+from structlog import get_logger
+
+
+class FTPService:
+    def __init__(
+        self,
+        host="localhost",
+        username="airflow",
+        password="airflow",
+        port=2222,
+        dir="/upload",
```

Comment:
maybe better to put it in config file (if you have one)?

Reply:
we don't have one, but I can write one :D

Reply:
`dynaconf` it's a great tool. Integrating would need changes in the current infrastructure (k8s), and I would suggest keeping reading the env vars for now.

Furthermore, `airflow` seems that provide similar functionality with `Variables` https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/variables.html that you can have it as configuration JSON per environment, which is similar to `dynaconf`.

Reply:
If we want to use the Variables from airflow uploaded by JSON file, we will need to change k8s as well, by importing the file in airflow: `airflow variables import properties.json`
I will put in the make file, for development